<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wooniezie Robot</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="credentialsBox_">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>165</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Credentials</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>21</y>
         <width>25</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
       <property name="buddy">
        <cstring>login_</cstring>
       </property>
      </widget>
      <widget class="QLineEdit" name="login_">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>40</y>
         <width>133</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>andrewst</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>66</y>
         <width>46</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
       <property name="buddy">
        <cstring>password_</cstring>
       </property>
      </widget>
      <widget class="QLineEdit" name="password_">
       <property name="geometry">
        <rect>
         <x>21</x>
         <y>85</y>
         <width>133</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Vfkbyf1%</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="loginButton_">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="0" rowspan="3">
     <widget class="QGroupBox" name="offerBox_">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>330</width>
        <height>400</height>
       </size>
      </property>
      <property name="title">
       <string>Offer Details</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Offer ID</string>
       </property>
      </widget>
      <widget class="QCalendarWidget" name="date_">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>296</width>
         <height>183</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Date</string>
       </property>
       <property name="buddy">
        <cstring>date_</cstring>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>290</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Time</string>
       </property>
       <property name="buddy">
        <cstring>time_</cstring>
       </property>
      </widget>
      <widget class="QTimeEdit" name="time_">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>310</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="clickNowButton_">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>350</y>
         <width>80</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Click Now</string>
       </property>
      </widget>
      <widget class="QPushButton" name="scheduleClickButton_">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>350</y>
         <width>80</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Schedule Click</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="offerId_">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>101</width>
         <height>22</height>
        </rect>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
       <property name="value">
        <number>51808</number>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="1" rowspan="4">
     <widget class="QWebEngineView" name="webView_">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>865</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
